ramp-up-web/public/index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+
+
+    <link rel="stylesheet" href="https://kendo.cdn.telerik.com/themes/5.6.0/material/material-main-dark.css"></link>
+    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    
+
+    <style>
```

Comment:
Don't put styles in the index.html file make a separate stylesheet and attach it to the index.js file